modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoMessage.java

```diff
@@ -222,24 +236,30 @@ boolean isOrdered() {
                 writer.incrementState();
 
             case 3:
-                if (!writer.writeBoolean("skipOnTimeout", skipOnTimeout))
+                if (!writer.writeUuid("secSubjId", secSubjId))
```

Comment:
Can we insert a new field into the middle of an existing message? What about backward compatibility?

Reply:
yes